code/src/terrat_files/github/sql/fail_running_work_manifest.sql

```diff
@@ -1,7 +1,10 @@
-update work_manifests as gwm
+update work_manifests as wm
 set state = 'aborted', completed_at = now()
 from github_installation_repositories as gir
-where gir.id = gwm.repository
+inner join github_work_manifests as gwm
+      on gwm.id = wm.id
+where gwm.id = wm.id
+      and gir.id = gwm.repository
```

Comment:
I'm not sure if this is strictly necessary but I know deletes with joins have some funky semantics so rather safe-than-sorry IMO.